src/table/TableWrapper.jsx

```diff
@@ -29,6 +34,90 @@ const useStyles = makeStyles({
   },
 });
 
+const useStyles2 = makeStyles({
+  root: {
+    flexShrink: 0,
+  },
+  disabled: {
+    color: 'rgba(0, 0, 0, 0.3)',
+    '&:hover': {
+
+    }
+  }
+});
+
+function TablePaginationActions(props) {
+  const classes = useStyles2();
+  const { count, page, rowsPerPage, onChangePage } = props;
+
+  const handleFirstPageButtonClick = (event) => {
+    onChangePage(event, 0);
+  };
+
+  const handleBackButtonClick = (event) => {
+    onChangePage(event, page - 1);
+  };
+
+  const handleNextButtonClick = (event) => {
+    onChangePage(event, page + 1);
+  };
+
+  const handleLastPageButtonClick = (event) => {
+    onChangePage(event, Math.max(0, Math.ceil(count / rowsPerPage) - 1));
```

Comment:
How could it be < 0? if there is only one page? then it should be disabled anyway I think?